tests/test_get_data.py

```diff
@@ -1,6 +1,10 @@
 import logging
 
 import pytest
+import io
+import os
+import threading
+from six import PY2
```

Comment:
We don't actually have an explicit dependency on six currently. It works because we get it transitively from `python-dateutil`, but that's not reliable (our direct deps could drop six at any time).  Probably better just to check `if sys.version_info.major < 3` or similar